pkg/install/v1alpha1/jiva_volume.go

```diff
@@ -657,13 +659,19 @@ spec:
     action: list
     options: |-
       labelSelector: openebs.io/replica=jiva-replica,openebs.io/persistent-volume={{ .Volume.owner }}
-    retry: "12,10s"
+    retry: "24,5s"
   post: |
     {{- jsonpath .JsonResult "{.items[*].metadata.name}" | trim | saveAs "createlistrep.items" .TaskResult | noop -}}
     {{- .TaskResult.createlistrep.items | empty | verifyErr "replica pod(s) not found" | saveIf "createlistrep.verifyErr" .TaskResult | noop -}}
     {{- jsonpath .JsonResult "{.items[*].spec.nodeName}" | trim | saveAs "createlistrep.nodeNames" .TaskResult | noop -}}
-    {{- $expectedRepCount := .Config.ReplicaCount.value | int -}}
-    {{- .TaskResult.createlistrep.nodeNames | default "" | splitList " " | isLen $expectedRepCount | not | verifyErr "number of replica pods does not match expected count" | saveIf "createlistrep.verifyErr" .TaskResult | noop -}}
+    {{- $expectedRepCount := div .Config.ReplicaCount.value 2 | add1 | int -}}
+    {{- .TaskResult.createlistrep.nodeNames | default "" | splitListLen " " | saveAs "createlistrep.noOfReplicas" .TaskResult | noop -}}
+    {{- if ge .TaskResult.createlistrep.noOfReplicas $expectedRepCount }}
```

Comment:
can it be something like:
- keep checking for replicacount number of replicas
- if reached replicacount, patch and exit
- if not reached but iterations are completed, check for consistency factor number of replicas
- if reached consistency factor, patch, warn and exit
- if not reached consistency factor, revert